cmd/dispatcher/main.go

```diff
@@ -112,6 +132,18 @@ func main() {
 		logger.Fatalf("Failed to create lock: %v", err)
 	}
 
+	calculationList, err := vegaClient.VegaV1().Calculations().List(context.TODO(), v1.ListOptions{})
+	if err != nil {
+		logger.WithError(err).Fatal(err)
+	}
+
+	for i := range calculationList.Items {
+		calculationValues.With(prometheus.Labels{"calc_id": calculationList.Items[i].Name, "status": string(calculationList.Items[i].Phase), "creation_time": calculationList.Items[i].Status.StartTime.Time.String()}).Inc()
```

Comment:
I don't think that this is the correct place to expose the metrics. Here we will expose the existing calculations once the dispatcher will first runs. Instead, we need the controllers to expose the metrics on every calculation change.

Reply:
Now when we get the resources of calculation in controller we expose the values.

Reply:
How can I trigger this piece of code? @droslean 

https://github.com/vega-project/ccb-operator/blob/27efeaec0a57e8e1477f478745613350959b7068/cmd/dispatcher/main.go#L124